go.mod

```diff
@@ -2,34 +2,28 @@ module go.dedis.ch/cothority/v4
 
 require (
 	github.com/BurntSushi/toml v0.3.1
-	github.com/allegro/bigcache v1.2.1 // indirect
 	github.com/aristanetworks/goarista v0.0.0-20191015010157-ce4ff66f8dba // indirect
 	github.com/bford/golang-x-crypto v0.0.0-20160518072526-27db609c9d03
 	github.com/btcsuite/btcd v0.20.0-beta // indirect
 	github.com/coreos/go-oidc v2.1.0+incompatible
-	github.com/deckarep/golang-set v1.7.1 // indirect
-	github.com/edsrzf/mmap-go v1.0.0 // indirect
 	github.com/ethereum/go-ethereum v1.8.27
-	github.com/hashicorp/golang-lru v0.5.3 // indirect
 	github.com/pkg/browser v0.0.0-20180916011732-0a3d74bf9ce4
-	github.com/pquerna/cachecontrol v0.0.0-20180517163645-1555304b9b35 // indirect
 	github.com/prataprc/goparsec v0.0.0-20180806094145-2600a2a4a410
 	github.com/qantik/qrgo v0.0.0-20160917134849-0c6b902c59f6
-	github.com/rs/cors v1.7.0 // indirect
 	github.com/satori/go.uuid v1.2.0
 	github.com/stretchr/testify v1.4.0
-	github.com/syndtr/goleveldb v1.0.0 // indirect
 	github.com/urfave/cli v1.22.0
+	go.dedis.ch/cothority/v3 v3.3.1
```

Comment:
You probably have a _build_ folder somewhere that is using v3. Please clean it and redo go mod tidy.

Reply:
old `build/conode_` directories from `test.sh`...